pkg/cmd/ssh/ssh_test.go

```diff
@@ -47,32 +47,20 @@ type bastionStatusPatch func(status *operationsv1alpha1.BastionStatus)
 func waitForBastionThenPatchStatus(ctx context.Context, gardenClient client.Client, bastionName string, namespace string, patcher bastionStatusPatch) {
 	defer GinkgoRecover()
 
-	ticker := time.NewTicker(1 * time.Second)
```

Comment:
hm not sure why it was done that complicated in the first place with the ticker 🤔

Reply:
Yes, I was wondering the same, because `Eventually` will block then retry by itself until the timeout is reached or the condition is met 😅